examples/01-basic/11-file-uploading-uppy/App.tsx

```diff
@@ -0,0 +1,70 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import "@blocknote/core/fonts/inter.css";
+import { BlockNoteView } from "@blocknote/mantine";
+import "@blocknote/mantine/style.css";
+import { useCreateBlockNote, FilePanelController } from "@blocknote/react";
+
+import Uppy from "@uppy/core";
+import XHR from "@uppy/xhr-upload";
+import { Dashboard } from "@uppy/react";
+
+import "@uppy/core/dist/style.min.css";
+import "@uppy/dashboard/dist/style.min.css";
+
+export default function App() {
+  // Creates a new editor instance.
+  const editor = useCreateBlockNote({
```

Comment:
could you also implement `uploadFile` as mentioned earlier?

Reply:
Uppy doesn't seem to support uploading files with custom function. So I don't think so we can make changes to uploadFile

Reply:
On it

Reply:
Seems like uppy don't support custom functions for file upload. I couldn't find any walkarounds